<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Countdown Timer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #1e1e1e;
    color: white;
    font-family: Arial;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0a3d62;</string>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>H:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="hourSpinBox">
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #2c3e50;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>M:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="minuteSpinBox">
        <property name="maximum">
         <number>59</number>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #2c3e50;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>S:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="secondSpinBox">
        <property name="maximum">
         <number>59</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #2c3e50;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #27ae60;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #e74c3c;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReset">
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #f39c12;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>